update_repos

```diff
@@ -207,6 +207,10 @@ class WikiRepo(CodeRepo):
         self.ssh_url = self.ssh_url.rpartition('.git')[0] \
                        + '.wiki.git'
         self.default_branch = 'master'
+
+        if self.description == None:
+            self.description = gh_repo.name
```

Comment:
This doesn't seem right. Since `WikiRepo` is derived from `CodeRepo`, then `self.description` will either be a real description or `""`. How about we always do this in `CodeRepo` and `JsonRepo`:
```
self.description = gh_repo.description or self.name
```
Then you don't need to worry about it in `WikiRepo`.